### DISCUSSION TITLE
Query about the I*R rate-heterogeneity across sites model

### DISCUSSION THREAD

**Discussion by lsjermiin** [iqtree/iqtree2#301] [Q&A]
Hi, 

I am using an alignment of amino acids to test the performance of ModelFinder (implemented in IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Apr 26 2024). 

In the present case, I used the following command:

`iqtree2 -s Aligned_proteins_concatenated.fst -st AA -m MF -mset mtVer -mrate *G,I*G,*R,I*R,*H -mfreq FU,FO,F -mtree -T 16 -merit AIC -safe`

The program aborted while processing the `mtVer+FO+I*R2` model (alignment and log file attached).

Is this because the `I*R` model has not been implemented or is there a bug in the code? 

Incidentally, the program completed its task under the BIC criterion.

Feedback would be most welcome.

Lars

[05_MF3_RHAS_AIC.zip](https://github.com/user-attachments/files/16636887/05_MF3_RHAS_AIC.zip)

**Comment by bqminh** [comment]
Sorry for the delay. From the log, it looks like +FO does not work with *G or *H. The likelihood doesn't make sense as it decreases during the optimisation. So please refrain from this combination of model.
